index.bs

```diff
@@ -185,7 +185,7 @@ Model {#model}
 The {{OrientationSensor}} class extends the {{Sensor}} class and provides generic interface
 representing device orientation data.
 
-The OrientationSensor has an associated {{PermissionName}} which is <a for="PermissionName" enum-value>"orientation-sensor"</a>.
+To access orientation sensor's [=latest reading=], user agent must invoke [=request sensor access=] abstract operation for each of the [=low-level=] sensors used by the concrete orientation sensor.
```

Comment:
@anssiko I added table, alignment is off :( but this is bikeshed bug, same happens with other specs.